test/test_foreach.py

```diff
@@ -14,7 +14,7 @@
     (instantiate_device_type_tests, dtypes, onlyCUDA, skipMeta, ops)
 from torch.testing._internal.common_methods_invocations import (
     foreach_unary_op_db, foreach_binary_op_db, foreach_pointwise_op_db, foreach_minmax_op_db,
-    foreach_reduce_op_db)
+    foreach_reduce_op_db, foreach_lerp_op_db)
```

Comment:
why do we need separate foreach_lerp_op_db, can it be part of foreach_pointwise_op_db (which, iirc, has 3 arguments).

Reply:
the number of arguments and their pattern can be different. pointwise, addcmul and addcdiv, take three lists of tensors and an optional scalar or an optional list of scalars while the arguments of lerp can be two lists of tensors and one list of tensors and one scalar.

Reply:
Regular ops covered by OpDB have very different arguments and yet are all tested by common tests, so I think it should be possible not to multiply dbs here

Reply:
After checking how the other `OpInfo` tests e.g. test_ops are doing, it seems like I should rely on `SampleInput` https://github.com/pytorch/pytorch/blob/b3206268ace6ebcb5d716ed6673876e62ef484f2/torch/testing/_internal/opinfo/core.py#L114 to better handles different arguments